lib/Parse/ParseIfConfig.cpp

```diff
@@ -331,6 +332,8 @@ class ValidateIfConfigCondition :
         DiagName = "architecture"; break;
       case PlatformConditionKind::Endianness:
         DiagName = "endianness"; break;
+      case PlatformConditionKind::CanImport:
+        DiagName = "import conditional"; break;
```

Comment:
Is this testable?

Reply:
As of now, no.  There's no validation of the module name being performed, so there's no way this will get hit.

Reply:
Okay, just checking.